tags_test.go

```diff
@@ -530,7 +530,7 @@ func TestUpdateCheckTags(t *testing.T) {
 	r, err := sc.UpdateCheckTags("11223344-5566-7788-9900-aabbccddeeff",
 		[]string{
 			"test:test",
-			"dGVzdA==:dGVzdA==",
+			"b\"dGVzdA==\":b\"dGVzdA==\"",
```

Comment:
out of curiosity, what happens if the `:` is b64 encoded (say a customer does this by accident)?

Reply:
Well, from the point of view of what is affected by this particular piece of code, you would have an invalid check tag and an error would be returned. Snowth check tags are required to have both a category and a value, separated by a colon that is not encoded. The code from Perl enforces that, but allows either the category or value to be blank. This raises a good question as a tag without a category is technically valid, and we may need to support that here. But, I need to check what snowth requires before adding that.